docs/react-wiki-archive/Build-Commands.md

```diff
@@ -41,7 +38,7 @@ Many of the repo-wide commands internally use the Lage task runner and support i
 |[`yarn update-snapshots`](#yarn-update-snapshots-repo)|Update Jest snapshots for all packages *(**must build first**; supports `--to`)*|
 |[`yarn scrub`](#yarn-scrub)|Revert the repo to a clean state *(commit or stash your changes first)*|
 |[`yarn create-package`](#yarn-create-package)|Create a new package|
-|`yarn lage run`|(advanced) Run any package.json script name repo-wide using Lage. This is usually unnecessary. *(supports `--to`)*|
+|`yarn nx run-many`|(advanced) Run any package.json script name repo-wide using nx. This is usually unnecessary. *(supports `--to`)*|
```

Comment:
Should the `(supports --to)` part be removed? I don't believe `run-many` or nx in general supports that

Reply:
good catch ! ty

Reply:
on second thought this is old wiki docs - I'm gonna revert changes for this file as it's no longer relevant.